drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -177,6 +210,9 @@ int spi_mcux_configure(const struct device *dev, const struct spi_config *spi_cf
 	LPSPI_MasterInit(base, &master_config, clock_freq);
 	LPSPI_SetDummyData(base, 0);
 
+	/* Always use continuous mode to satisfy SPI API requirements. */
+	base->TCR |= LPSPI_TCR_CONT_MASK;
+
```

Comment:
the handling of the transfer should be done in the transfer driver files, not the common file. The only acceptable writing of the TCR currently should be for configuring aspects coming from the zephyr api spi_config struct.  This breaks the abstraction of the spi_mcux_configure function, which is not supposed to assume anything about the logical implementation and handling of the transfer beyond what is specified in the configuration of the api. So request is to move this line out of this function and out of this file.

Reply:
to my understanding, continuous mode(keep PCS asserted during a SPI transfer) is a common requirement of SPI API.
let's discuss it offline in detail.